octo-core/lib/octocore/models/enterprise/funnel_tracker.rb

```diff
@@ -0,0 +1,14 @@
+require 'cequel'
+
+module Octo
+  class FunnelTracker
```

Comment:
A few comments about the tracker class and the data structure it encapsulates. Also this class should provide helper functions like `incoming_nodes`, `outgoing_nodes` , `load_factor` etc. These would be helpful in computing markov chains later.